### PR TITLE
dev-aws/cbc: add rating events tls config

### PR DESCRIPTION
**Pull request by LukeG07**

### CODE REVIEW

File: dev-aws/kafka-shared-msk/cbc/cbc.tf [hunk 1]
```diff
@@ -16,11 +16,30 @@ resource "kafka_topic" "FraudEvents" {
   }
 }
 
+# tflint-ignore: terraform_naming_convention
+resource "kafka_topic" "RatingEvents_v3" {
+  name = "cbc.RatingEvents_v3"
+
+  replication_factor = 3
+  partitions         = 15
```
Comment (sbuliarca) [inline dev-aws/kafka-shared-msk/cbc/cbc.tf:24]: Is this a migrated topic? If not, can it manage with less partitions?

Comment (martskins) [reply dev-aws/kafka-shared-msk/cbc/cbc.tf:24]: it is a migrated topic yeah, we'll be migrating a bunch of them in the next few days

File: dev-aws/kafka-shared-msk/cbc/cbc.tf [hunk 2]
```diff
@@ -16,11 +16,30 @@ resource "kafka_topic" "FraudEvents" {
   }
 }
 
+# tflint-ignore: terraform_naming_convention
```
Comment (sbuliarca) [inline dev-aws/kafka-shared-msk/cbc/cbc.tf:19]: Can you actually fix the naming, please, since this is a new resource?

Comment (sbuliarca) [reply dev-aws/kafka-shared-msk/cbc/cbc.tf:19]: This is about the name of TF resource, not the name of the topic

Comment (LukeG07) [reply dev-aws/kafka-shared-msk/cbc/cbc.tf:19]: Ah I was under the impression that it had to match the name of the source topic, is that not the case?

Comment (sbuliarca) [reply dev-aws/kafka-shared-msk/cbc/cbc.tf:19]: Nope, they are separate things and they don't need to match

Comment (LukeG07) [reply dev-aws/kafka-shared-msk/cbc/cbc.tf]: Okay, good to know. That's it fixed

File: dev-aws/kafka-shared-msk/cbc/cbc.tf [hunk 3]
```diff
@@ -16,11 +16,29 @@ resource "kafka_topic" "FraudEvents" {
   }
 }
 
+resource "kafka_topic" "rating_rvents_v3" {
```
[the remark this answers is not in the document]
Comment (LukeG07) [reply dev-aws/kafka-shared-msk/cbc/cbc.tf]: My bad. Fixed

File: dev-aws/kafka-shared-msk/cbc/cbc.tf [hunk 4]
```diff
@@ -16,11 +16,30 @@ resource "kafka_topic" "FraudEvents" {
...
+resource "kafka_topic" "RatingEvents_v3" {
+  name = "cbc.RatingEvents_v3"
+
+  replication_factor = 3
+  partitions         = 15
+
+  config = {
+    "remote.storage.enable" = "true"
+    "retention.bytes"       = "-1"
+    "retention.ms"          = "-1"
```
[the remark this answers is not in the document]
Comment (LukeG07) [reply dev-aws/kafka-shared-msk/cbc/cbc.tf:28]: I think there was a similar question raised in our last PR for FraudEvents and my answer would be similar. We'd prefer to keep the config as similar to our old cluster as possible until we have the time to investigate the impact optimising these settings might have.

[the remark this answers is not in the document]
Comment (sbuliarca) [reply dev-aws/kafka-shared-msk/cbc/cbc.tf:28]: I just needed to ask :)

**Review by sbuliarca — APPROVED** [review]

**Pull request merged**